spec/views/registered_applications/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
You can remove all of these view specs.

Reply:
I thought rails_helper is needed to run the tests?